kiosk/src/Models/Kiosk.ts

```diff
@@ -180,7 +221,9 @@ export class Kiosk {
             gamespace.firstChild.remove();
         }
 
-        const playUrl = `${configData.PlayUrlRoot}?id=${gameId}&hideSimButtons=1&noFooter=1&single=1&fullscreen=1&autofocus=1`;
+        const playUrlBase = `${configData.PlayUrlRoot}?id=${gameId}&hideSimButtons=1&noFooter=1&single=1&fullscreen=1`
```

Comment:
autofocus went away? that one's a bit important I believe as it's necessary to give focus on game load (otherwise controllers won't necessarily be read properly )

Reply:
Oh, this was a mistake. I was trying to play with some of the flags to see if they changed anything. I will add it back.